### PR TITLE
allow Close button in Modal to be hidden

### PR DESCRIPTION
**Pull request by andrewharvey**

### CODE REVIEW

File: lib/components/modal.vue [hunk 1]
```diff
@@ -48,7 +48,7 @@
 
                         <footer class="modal-footer" ref="footer" v-if="!hideFooter">
                             <slot name="modal-footer">
-                                <b-btn variant="secondary" @click="hide(false)">{{closeTitle}}</b-btn>
+                                <b-btn variant="secondary" @click="hide(false) v-if="!hideClose">{{closeTitle}}</b-btn>
```
Comment (tmorehouse) [inline lib/components/modal.vue]: Missing a closing quote on `@click="hide(false)`

File: lib/components/modal.vue [hunk 2]
```diff
@@ -161,6 +161,10 @@
                 type: Boolean,
                 default: false
             },
+            hideClose: {
```
Comment (tmorehouse) [inline lib/components/modal.vue]: I wonder if this would be better called 'okOnly'?

**Comment by andrewharvey** [comment]
> Are you thinking of this for use case as an alert (with only OK button)?

Exactly.

I'll make the 2 changes you've suggested. okOnly is probably better.

**Comment by tmorehouse** [comment]
Cool.

I'll let @pi0 review. He should be back online in a few hours.

**Comment by pi0** [comment]
Thanks!

**Pull request merged**